### PR TITLE
Removed a file which causes lots of issues on Unix

### PR DESCRIPTION
**Pull request by meepen**

### CODE REVIEW

**Comment by LennyPenny** [comment]
what is that wrap stuff

**Comment by mcd1992** [comment]
Don't store code in comments meep.

**Comment by meepen** [comment]
It's not stored, it's there in case anyone wants to fix it up.

**Comment by meepen** [comment]
AY YOU LITTLE NIGERIANS UNBAN ME FROM CHAT

**Comment by bmwalters** [comment]
nope

**Comment by meepen** [comment]
rood

**Comment by bmwalters** [comment]
sorry ily

**Comment by meepen** [comment]
:(

**Comment by bmwalters** [comment]
7:48 PM - Zerf: hey can you guys unban meep please

**Comment by meepen** [comment]
ty

File: Tier1/premake5.lua [hunk 1]
```diff
@@ -52,9 +52,22 @@ if os.is "windows" then
 	links { "Ws2_32", "Rpcrt4" }
 	files "../src/tier1/processor_detect.cpp"
 else
+
+--[[
```
Comment (mcd1992) [inline Tier1/premake5.lua:56]: >storing code in a comment when its already revisioned by git

Comment (PotcFdk) [inline Tier1/premake5.lua:56]: To some, version control is a hard concept, ok?

**Comment by SpencerSharkey** [comment]
:+1: fix
:-1: comments

File: Tier1/premake5.lua (hunk 1 above)
Comment (meepen) [inline Tier1/premake5.lua:56]: ur a version control

on a side note ur mom!!

Comment (SwadicalRag) [inline Tier1/premake5.lua:56]: pls
pls
:+1:

:100:-

**Comment by meepen** [comment]
overridden by #3

**Pull request closed without merging**